README.md

```diff
@@ -389,6 +390,23 @@ Running the build
     and you are behind a non-transparent proxy server,
     [configure your proxy server in Maven](http://maven.apache.org/settings.html#Proxies).
 
+ Note: if you get the following error: 
+ 
+    ```shell
+    [ERROR] Failed to execute goal org.codehaus.mojo:jaxb2-maven-plugin:1.5:schemagen (generate-xsd) on project kie-remote-jaxb-gen: Failed to generate schema: InvocationTargetException: Class not foundcom/sun/tools/apt/Main.class -> [Help 1]
+    ``` 
```

Comment:
This should have been fixed by https://github.com/droolsjbpm/droolsjbpm-integration/commit/73d1130f3921df16ed8ab2ee98baff87f5868a75. I just tried locally and everything works for me with Oracle JDK 1.8.0_51. If the problem persist, please let us know more details. We need to fix this issue, instead of providing workarounds.